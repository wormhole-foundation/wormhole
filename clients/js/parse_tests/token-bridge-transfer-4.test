010000000001001565b62bbf9978b1f9183ae985eb34f664fd4c850ec4ad8a38533281aec75eba2456f91c9a967cf4a70901aa0afed17ba39f1d779089b32eb88a47f7ea405e4b00000000002605c517000195f83a27e90c622a98c037353f271fd8f5f57b4dc18ebf5ff75a934724bd04914e4eb08ee374efbd200100000000000000000000000000000000000000000000000000000000000186a0165809739240a0ac03b98440fe8985548e3aa683cd0d4d9df5b5659669faa3010001000000000000000000000000c10820983f33456ce7beb3a046f5a83fa34f027d0c200000000000000000000000000000000000000000000000000000000000000000