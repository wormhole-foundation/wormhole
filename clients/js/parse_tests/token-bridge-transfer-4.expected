{
  version: 1,
  guardianSetIndex: 0,
  signatures: [
    {
      guardianSetIndex: 0,
      signature: '1565b62bbf9978b1f9183ae985eb34f664fd4c850ec4ad8a38533281aec75eba2456f91c9a967cf4a70901aa0afed17ba39f1d779089b32eb88a47f7ea405e4b00'
    }
  ],
  timestamp: 0,
  nonce: 637912343,
  emitterChain: 1,
  emitterAddress: '0x95f83a27e90c622a98c037353f271fd8f5f57b4dc18ebf5ff75a934724bd0491',
  sequence: 5642641510889746365n,
  consistencyLevel: 32,
  payload: {
    module: 'TokenBridge',
    type: 'Transfer',
    amount: 100000n,
    tokenAddress: '0x165809739240a0ac03b98440fe8985548e3aa683cd0d4d9df5b5659669faa301',
    tokenChain: 1,
    toAddress: '0x000000000000000000000000c10820983f33456ce7beb3a046f5a83fa34f027d',
    chain: 3104,
    fee: 0n
  },
  digest: '0x559318082f6abb8b0fcf360d2a98be84a0ccf6602044882cc0d6764a374ae44d'
}
